### PR TITLE
Level select

### LINKED ISSUE #194: level select design

### PR DESCRIPTION
**Pull request by mt-xing**
### Summary 

There's a new level select screen now.
Close #194

### Testing 

image

### Review Focus 

Changes to Main Menu Mode and Level Constants, mainly. The scene graph changes are dumb.

### Additional Issues 

Rotation doesn't work for some reason. I've asked on Piazza; we'll see what Walker says.

### CODE REVIEW

**Comment by wenjiazhang** [comment]
Can you make the easy/hard text smaller? I think it will look better

**Comment by wenjiazhang** [comment]
Maybe make the text like a blue color too

**Comment by mt-xing** [comment]
> Maybe make the text like a blue color too

I can make it smaller no problem. You're gonna have to pick the shade of blue though. I suck at picking colors.

**Comment by mt-xing** [comment]
image

@wenjiazhang Made it smaller.

**Comment by wenjiazhang** [comment]
Try (80, 117, 196) blue on blue is fine as long as the value has contrast

**Comment by mt-xing** [comment]
> Try (80, 117, 196) blue on blue is fine as long as the value has contrast

Eh...
image

Accessibility guidelines generally ask for 4.5:1 at the minimum; we're close, but not quite there.

**Comment by mt-xing** [comment]
image
I can bump the tint just a bit to get to 4.5; does this look okay?

**Comment by mt-xing** [comment]
image

File: cugl/lib/2d/CUNode.cpp [hunk 1]
```diff
@@ -221,7 +221,7 @@ bool Node::initWithData(const SceneLoader* loader, const std::shared_ptr<JsonVal
     if (data->has("angle")) {
         transform = true;
         _angle = data->getFloat("angle",0.0f);
-        _angle = M_PI/180.0f;
+        _angle *= M_PI/180.0f;
```
Comment (mt-xing) [inline cugl/lib/2d/CUNode.cpp:224]: You better believe I'm mentioning this in the postmortem today

Comment (ssorenson130) [reply cugl/lib/2d/CUNode.cpp:224]: cugl? more like bugl amirite

**Pull request merged**